README.md

```diff
@@ -58,8 +58,8 @@ Stay up-to-date with KDAB product announcements:
 
 If you want to contribute, please check out: <https://github.com/KDAB/GammaRay/wiki/Get-Involved>
 
-KDAB will happily accept external contributions, but substantial contributions require
-a signed [Copyright Assignment Agreement](docs/GammaRay-CopyrightAssignmentForm.pdf).
+KDAB will happily accept external contributions, but all contributions require
+agreeing to our Copyright Assignment Agreement.
```

Comment:
Is it worth mentioning here, that a bot will check PRs when opened and provide a link to the online CLA form if necessary?